app/models/reports/performance_score.rb

```diff
@@ -0,0 +1,49 @@
+module Reports
+  class PerformanceScore
+    CONTROL_SCORE_WEIGHT = 0.5
+    VISITS_SCORE_WEIGHT = 0.3
+    REGISTRATIONS_SCORE_WEIGHT = 0.2
+    TARGET_REGISTRATION_RATE = 0.1
+
+    def initialize(region:, result:)
```

Comment:
Should we call this `reports_result`? Took me a while to figure out what `result` this is.

Reply:
I definitely don't want to confuse; renamed it to `reports_result`